pep-0012.rst

```diff
@@ -50,44 +50,36 @@ To use this template you must first decide whether your PEP is going
 to be an Informational or Standards Track PEP.  Most PEPs are
 Standards Track because they propose a new feature for the Python
 language or standard library.  When in doubt, read PEP 1 for details
-or contact the PEP editors <peps@python.org>.
+or contact a PEP editor for assistance.
```

Comment:
Does PEP 12 somewhere give instructions on how to contact a PEP editor?  I can't find that in the text jigsaw of Github's diff browser.

Reply:
Not currently. Mariatta also asked about that, so there's a subthread above.